package.json

```diff
@@ -22,6 +22,7 @@
     "@mantine/hooks": "7.13.0",
     "@mantine/notifications": "^7.13.0",
     "@nivo/sankey": "^0.87.0",
+    "@radix-ui/react-icons": "^1.3.1",
```

Comment:
(So we can remove this dep)

Reply:
Kinda make sense, if there already is a icon lib. I just use the example one from React flow